src/e2e/specs/auth.spec.ts

```diff
@@ -23,15 +24,20 @@ test.describe(`${process.env.E2E_TEST_ENV} - Authentication flow verification @s
     await landingPage.goToSignIn();
 
     // Fill out sign up form
-    const randomEmail = `${Date.now()}_tstact@restmail.net`;
+    const currentTimestamp = Date.now();
+    const randomEmail = `${currentTimestamp}_tstact@restmail.net`;
     await authPage.signUp(randomEmail, page);
 
     // assert successful login
-    const successUrl =
-      process.env.E2E_TEST_ENV === "local"
-        ? "/user/dashboard"
-        : "/user/welcome";
-    expect(page.url()).toBe(`${process.env.E2E_TEST_BASE_URL}${successUrl}`);
+    const successUrlSlugs = isPrePlusDate(
+      process.env.BROKER_SCAN_RELEASE_DATE ?? "",
+      new Date(currentTimestamp).toUTCString(),
+    )
+      ? "/user/dashboard"
```

Comment:
I think we now never run this branch, right? Do we need a test that explicitly hits this branch, or alternatively, should we just remove it?

Reply:
We can [319fc47](https://github.com/mozilla/blurts-server/pull/4996/commits/319fc47be50bdcec6761df1eaade75989e25c3c4) this part for now. We’ve only tested one of the paths before (basically forcing the pre-Plus one). Covering both cases would be a good addition, but a task for when we are extending the tests.